contrib/mesos/pkg/minion/tasks/task.go

```diff
@@ -29,7 +29,10 @@ import (
 	"k8s.io/kubernetes/contrib/mesos/pkg/runtime"
 )
 
-const defaultTaskRestartDelay = 5 * time.Second
+const (
+	defaultTaskRestartDelay = 5 * time.Second
+	defaultKillGracePeriod  = 5 * time.Second
```

Comment:
What is the timeout we are granted by Mesos? Things might get dirty when that one is lower than the above, I guess.